// protos/user.proto

syntax = "proto3";

package user;

service UserService {
  rpc CreateUser (UserRequest) returns (UserReply);
  rpc GetUser (UserIdRequest) returns (UserReply);
}

message UserRequest {
  string name = 1;
  string email = 2;
}

message UserIdRequest {
  int32 id = 1;
}

message UserReply {
  int32 id = 1;
  string name = 2;
  string email = 3;
}
